Source/Core/Core/HW/EXI/EXI_DeviceIPL.cpp

```diff
@@ -158,6 +158,7 @@ void CEXIIPL::DoState(PointerWrap& p)
   p.Do(m_cursor);
   p.Do(m_buffer);
   p.Do(m_fonts_loaded);
+  p.Do(g_SRAM);
```

Comment:
I'm guessing we don't need the `p.Do(g_SRAM.rtc);` above anymore?

Reply:
ping

Reply:
Yeah you're right, I'll fix that later.